client/src/components/History/Content/ContentItem.vue

```diff
@@ -94,6 +94,11 @@ const jobState = computed(() => {
     return new JobStateSummary(props.item);
 });
 
+const itemIsRunningInteractiveTool = computed(() => {
+    // If our datset id is in the entrypOintStore it's a running it
+    return entryPointStore.entryPointsForHda(props.item.id).length > 0;
```

Comment:
I think this can be a collection or a dataset ? (unfortunate imo).
```suggestion
    return !isCollection.value && entryPointStore.entryPointsForHda(props.item.id).length > 0;
```